Test/AwsCommonRuntimeKitTests/mqtt/Mqtt5ClientTests.swift

```diff
@@ -843,17 +930,20 @@ class Mqtt5ClientTests: XCBaseTestCase {
             XCTAssertEqual(negotiatedSettings.maximumQos, QoS.atLeastOnce)
         } else {
             XCTFail("Missing negotiated settings")
+            return
```

Comment:
Also per discussion, feels like there's a tremendous amount of boilerplate repeated across different sets of tests.  Much broader that existing review obviously.